install/kubernetes/helm/istio/values.yaml

```diff
@@ -297,6 +297,33 @@ global:
     # If set to true, the pilot and citadel mtls and the plain text pilot ports
     # will be exposed on an internal gateway
     useILB: false
+  # Configure the mesh networks to be used by the Split Horizon EDS.
+  # Please notice that it is an Alpha level feature.
+  #
+  # The following example defines two networks with different endpoints
+  # association methods. For `network1` all endpoints that their IP belongs
+  # to the provided CIDR range will be mapped to network1. The gateway for
+  # this network example is specified by its public IP address and port.
+  # The second network, `network2`, in this example is defined differently
+  # with all endpoints retrieved through the specified Multi-Cluster registry
+  # being mapped to network2. The gateway is also defined differently with
+  # the name of the gateway service on the remote cluster. The public IP for
+  # the gateway will be determined from that remote service (not supported
+  # yet).
+  # networks:
+  #   network1:
+  #     endpoints:
+  #     - fromCidr: "192.168.0.1/24"
+  #     gateways:
+  #     - address: 1.1.1.1
+  #       port: 80
+  #   network2:
+  #     endpoints:
+  #     - fromRegistry: reg1
+  #     gateways:
+  #     - registryServiceName: istio-ingressgateway
+  #       port: 443
+  networks: {}
```

Comment:
same here as well. @ymesika had them and them moved them. So, it would be good to tacke this helm stuff in separate PR perhaps